src/components/IOUConfirmationList.js

```diff
@@ -80,6 +85,11 @@ const defaultProps = {
 };
 
 class IOUConfirmationList extends Component {
+    constructor(props) {
+        super(props);
+        this.minimumBottomOffset = 240;
```

Comment:
Does 240 represent the size of the button area? I think this will change in the future, so would we need to make this dynamic in that case?

Also, I don't think it needs to be attached to IOUConfirmationList. Could you make this a constant please?

Reply:
It was based on hit and trial to accommodate the button as well as the comment box. Will include it in CONST.js then! :)

Reply:
But, isn't it more appropriate in IOUConfirmationList itself, considering it includes both the Button and Comment Box, which is only present in IOUConfirmationList right now? @Julesssss

Reply:
I mean more that it doesn't need to be a class prop. A const within this file would be fine too I think.

Reply:
@Julesssss Ahh makes sense! Made the change :)